docs/user/guides/environment-variables.rst

```diff
@@ -0,0 +1,87 @@
+How to use custom environment variables
+=======================================
+
+If extra environment variables are needed in the build process,
+you can define them from the project's :term:`dashboard`.
+
+.. seealso::
+
+   :doc:`/environment-variables`
+     Learn more about how Read the Docs applies environment variables in your builds.
+
+Go to your project's :menuselection:`Admin --> Environment Variables` and click on :guilabel:`Add Environment Variable`.
+You will then see the form for adding an environment variable:
+
+.. image:: /img/screenshot_environment_variables.png
+   :alt: Screenshot of the form for adding an environment variable
+
+#. Fill in the :guilabel:`Name` field, this is the name of your variable, for instance ``SECRET_TOKEN`` or ``PIP_EXTRA_INDEX_URL``.
+#. Fill in the :guilabel:`Value` field with the environment variable's value, for instance a secret token or a build configuration.
+#. Check the :guilabel:`Public` option if you want to expose this environment variable
+   to :doc:`builds from pull requests </pull-requests>`.
+
+   .. warning::
+
+      If you mark this option, any user that can create a pull request
+      on your repository will be able to see the value of this environment variable.
+      In other words,
+      do not use this option if your environment variable is a secret.
+
+.. note::
```

Comment:
Don't love 2 callouts back to back, but seems OK in this case? Not sure this one needs a note vs. just normal text.

Reply:
It was fairly easy to convert it to normal text, yes :+1: